.teamcity/_self/projects/WPComTests.kt

```diff
@@ -222,50 +215,88 @@ fun editorTrackingBuildType( targetDevice: String, buildUuid: String, atomic: Bo
 	)
 }
 
-fun jetpackPlaywrightBuildType( targetDevice: String, buildUuid: String, jetpackTarget: String = "wpcom-production" ): E2EBuildType {
-	val idSafeJetpackTarget = jetpackTarget.replace( "-", "_" );
-	val targetJestGroup = if (jetpackTarget == "remote-site") "jetpack-remote-site" else "jetpack-wpcom-integration";
+fun jetpackSimpleDeploymentE2eBuildType( targetDevice: String, buildUuid: String ): BuildType {
+	return BuildType({
+		id("WPComTests_jetpack_simple_deployment_e2e_$targetDevice")
+		uuid = buildUuid
+		name = "Jetpack Simple Deployment E2E Tests ($targetDevice)"
+		description = "Runs E2E tests validating the deployment of Jetpack on Simple sites on $targetDevice viewport"
+		
+		artifactRules = defaulE2eArtifactRules();
+
+		vcs {
+			root(Settings.WpCalypso)
+			cleanCheckout = true
+		}
+
+		params {
+			defaultE2eParams()
+			calypsoBaseUrlParam()
+			param("env.VIEWPORT_NAME", "$targetDevice")
+			param("env.JETPACK_TARGET", "wpcom-deployment")
+		}
 
-	val triggers: Triggers.() -> Unit = {
-		if (jetpackTarget == "wpcom-staging") {
+		triggers {
 			finishBuildTrigger {
 				buildType = "JetpackStaging_JetpackSunMoonUpdated"
 			}
-		} else {
-			// For remote-site tests and production, we are just running daily for now.
-			schedule {
-				schedulingPolicy = daily {
-					hour = 5
+		}
+
+		steps {
+			prepareE2eEnvironment()
+
+			bashNodeScript {
+				name = "Run tests"
+				scriptContent = """
+					# Configure bash shell.
+					shopt -s globstar
+					set -x
+
+					# Enter testing directory.
+					cd test/e2e
+					mkdir temp
+
+					# Disable exit on error to support retries.
+					set +o errexit
+
+					# Run suite.
+					xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=jetpack-wpcom-integration
+
+					# Restore exit on error.
+					set -o errexit
+
+					# Retry failed tests only.
+					xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=jetpack-wpcom-integration --onlyFailures
+				"""
+				dockerImage = "%docker_image_e2e%"
+			}
```

Comment:
This step is the next piece for future modularization! I'd love to make some kind of builder or smart library function that makes it easy to build bash scripts that auto-handle a lot of shared configuration, and things like retries. But, I want to see how those seams play out when we need to do our atomic variation looping! So I'll likely add that abstraction then.